src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -18,6 +18,11 @@
     <PackageReference Include="System.Memory" Version="4.5.1" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.1" />
     <PackageReference Include="System.Text.Encoding" Version="4.3.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Experimental.Collections\Microsoft.Experimental.Collections.csproj" />
```

Comment:
Do we really need to be dependent on this experimental library? Is it planning on shipping?

/cc @safern

Reply:
I think he depends on MultiValueDictionary... the plan for experimental collections is that if a collection gets popular enough and might be useful for partners or the framework, we will move that collection into the product.

Reply:
I think MultiValueDictionary already got formally API reviewed (as part of Ian's internship). That was before my time, but assuming so then most likely it would only need a brief re-review. And I think it should likely go into CoreFX for 5.0. It seems a fairly basic, uncontroversial type.

Reply:
Is the plan for MultiValueDictionary to support `netstandard2.0` or lower?

Reply:
IMHO - for this case, using `Dictionary<TKey, ICollection<long>>` (or similar) would suffice, and it would alleviate the need for an added dependency. Making `DataFrame` dependent on a non-stable library affects how and when we could ship it in the future.

Reply:
Ah, I was worried this would be the case. I can change to using a `Dictionary`.